Update terminology frontend boilerplate docs - Closes 316 in lisk-sdk-examples (#1770)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc

const { apiClient } = require('@liskhq/lisk-client');

(async () => {

> 	// Update the path to point to your blockchain client data folder

Author: ```suggestion
	// Update the path to point to your blockchain client data folder for the respective sidechain
```

> 	// Update the path to point to your blockchain client data folder for the respective sidechain
> 	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
> 	const sidechainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');

Author: ```suggestion
	// Update the path to point to your sidechain client data folder
	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
```

	// Get active validators from sidechain

> 	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });

Reviewer: Yes, that sounds good..I think to further enhance the clarity in this statement, imho I would suggest the following:
"// Update the path to point to your blockchain client data folder for the respective sidechain"

> 	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });

Reviewer 2: I think this is unnecessary complicated. In this specific context of sidechain/mainchain communication, it still makes sense to talk about sidechain and mainchain nodes and blockchain clients. Also, if you want to be 100% precise here, you are connecting to a node, not the client. In the end, we would also need to update this in the code and I don't think it is necessary. Please just revert the changes.

> 	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });

Author: Changes reverted.

> 	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });

Author: ```suggestion
	const bftParams = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```
